lib/internal/fs/utils.js

```diff
@@ -716,6 +717,14 @@ function warnOnNonPortableTemplate(template) {
   }
 }
 
+const defaultCopyOptions = {
+  dereference: false,
+  errorOnExist: false,
+  overwrite: true,
```

Comment:
Would it be better to default to safer options? With `rm` the `force` and `recursive` options both default to `false`.

Reply:
@RyanZim's I'd be curious to know fs-extra's reasoning on the default option of overwrite, I'm inclined to agree with @iansu, but might be missing something.

Reply:
Reasoning? Because it's always been that way. :grinning:

Way back, fs-extra depended on https://github.com/AvianFlu/ncp for `copy`; and it overwrote by default, with no option, until https://github.com/AvianFlu/ncp/pull/19 added `overwrite`, then called `clobber`. They kept backwards compatibility by defaulting it to `true`.

Somewhere along the line, we inlined `ncp` into `fs-extra` since it was unmaintained, renamed `clobber` to `overwrite`, and added the `errorOnExist` option.

Then @manidlou rewrote our `copy` implementation in https://github.com/jprichardson/node-fs-extra/pull/502, since `ncp` was a painful legacy piece of code, but kept all the options the same. So, there's never been any real reasoning put into this.

Reply:
I've done two things:

* I've switched `overwrite` to being the parameter `force`, which feels more consistent with `rm` in Node.js, and the unix `cp` tool.
* I've defaulted `force` to `false`, as @iansu suggests.